CMake/macros.cmake

```diff
@@ -120,15 +120,31 @@ FUNCTION( RUN_QMC_APP_NO_COPY TESTNAME WORKDIR PROCS THREADS TEST_ADDED ${ARGN}
             MESSAGE("Disabling test ${TESTNAME} (building without MPI)")
         ENDIF()
     ENDIF()
+    IF ( TEST_ADDED )
```

Comment:
I think this should be ${TEST_ADDED} - since it is an output variable, references in the code need to use the name of the variable in TEST_ADDED, not TEST_ADDED itself.